lib/fluent/plugin/in_tail.rb

```diff
@@ -212,6 +213,20 @@ def configure(conf)
       @metrics = MetricsInfo.new(opened_file_metrics, closed_file_metrics, rotated_file_metrics, throttling_metrics)
     end
 
+    def check_dir_permission
+      expand_paths_raw.select { |path|
+        if !File.exist?(path)
```

Comment:
```suggestion
      expand_paths_raw.select { |path|
        not File.exist?(path)
      }.each { |path|
```

Sorry for my late response!

It would be more natural to separate `select` like this, or use `each` from the beginning.

Reply:
@daipom Fixed!